edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/KubernetesModule.cs

```diff
@@ -236,12 +236,16 @@ protected override void Load(ContainerBuilder builder)
                             this.proxyTrustBundlePath,
                             this.proxyTrustBundleVolumeName,
                             this.proxyTrustBundleConfigMapName,
+                            this.defaultMapServiceType,
                             this.persistentVolumeName,
                             this.storageClassName,
+                            this.persistentVolumeClaimSizeMb,
                             this.apiVersion,
                             this.workloadUri,
                             this.managementUri,
-                            this.runAsNonRoot))
+                            this.runAsNonRoot,
+                            this.enableServiceCallTracing,
+                            this.experimentalFeatures.GetEnvVars()))
```

Comment:
Really don't like it here, but cannot come up with better approach :(

Reply:
Let's make another class ExperimentalFeatures in Kubernetes project with one flag for now and make KubernetesExperimentalFeatures return ExperimentalFeatures. 
ExperimentalFeatures will have only flags that show experimental feature (only CreateOptions so far) and be passed everywhere it needed instead individual bool values.
KubernetesExperimentalFeatures will deal with reading values from IConfiguration and will stay in Service project.
```csharp
class ExperimentalFeatures {
  bool CreateOptions { get; }
}
```

Reply:
I was trying to avoid having the Deployment mapper know what features edgeAgent supports.  Even if I create another class, in the Kubernetes project, I'm still having that class create the environment vars.